lisa/features/gpu.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from enum import Enum
+from typing import Any, cast
+
+from lisa.feature import Feature
+from lisa.operating_system import Linux, Redhat, Ubuntu
+from lisa.tools import Uname, Wget
+from lisa.util import LisaException
+
+FEATURE_NAME_GPU = "Gpu"
+
+
+class compute_sdk(Enum):
+    GRID = 0
+    CUDA = 1
+
+
+class Gpu(Feature):
+    def __init__(self, node: Any, platform: Any) -> None:
```

Comment:
I guess we should fix these Any/Any on upper classes, this is ugly :(

Reply:
hmm, This is heavily used in the whole V3 code. Should take it up as a separate work to think about fixing it and clean it up all